analyses/cellranger-analysis/j2.sh

```diff
@@ -28,5 +28,4 @@ mkdir -p ./results/03_cellranger_count_summary
 ########################################################################
 # Run to summarize results generated by CellRanger for all libraries
 python ./util/summarize_cellranger_results.py --dir=./results/02_cellranger_count/${cellranger_parameters} \
-                                              --outdir=./results/03_cellranger_count_summary/ \
-                                              --genome ${genome_name_cellranger}
\ No newline at end of file
+                                              --outdir=./results/03_cellranger_count_summary/
```

Comment:
Maybe consider leaving the --genome flag within the script, but commented out (similar to how the --force_cells flag is left in the j1.sh script), to make it a bit simpler for the user to edit

Reply:
```suggestion
                                              --outdir=./results/03_cellranger_count_summary/
                                              # --genome ${genome_name_cellranger}
```

Reply:
See e71701a6671ae2b83fbb04835c926f2167c96b59